BUILD_INTERFACE.md

```diff
@@ -0,0 +1,34 @@
+Building an Interface for the Deckhandler
```

Comment:
for the file name, I thought "SWIG_INTERFACE.md" might be better, since it's specifically about swig and not for interfacing with a C app.